swc/private/swc.bzl

```diff
@@ -189,7 +189,7 @@ def _impl(ctx):
     plugin_args = []
     if ctx.attr.plugins:
         plugin_cache = [ctx.actions.declare_directory("{}_plugin_cache".format(ctx.label.name))]
-        plugin_args = ["--config", json.encode({
+        plugin_args = ["--config-json", json.encode({
```

Comment:
this will be confusing for anyone who already started using the feature. Maybe that's approx. no one so it's fine.

In the future when such changes happen we should just be conditional - we know what version of swc is being used so we can pass the correct flags for that version of the CLI.

Reply:
Yeah, I figured in this case it'd be fine, but I agree completely.